charts/stable/immich/Chart.yaml

```diff
@@ -24,9 +24,9 @@ maintainers:
     url: https://truecharts.org
 name: immich
 sources:
-  - https://github.com/truecharts/charts/tree/master/charts/incubator/immich
+  - https://github.com/truecharts/charts/tree/master/charts/stable/immich
```

Comment:
For the future you can skip those :P
I've had made a script to auto update it on push ;)

Save your precious seconds lol

Reply:
What will I ever do with those 5 seconds lol